code/model/editableformfields/EditableFormField.php

```diff
@@ -984,4 +1024,73 @@ public function EffectiveDisplayRules()
         }
         return $this->DisplayRules();
     }
+
+	/**
+	 * @return array|null
+	 */
+	public function formatDisplayRules()
+	{
+		$field = $this;
```

Comment:
Agreed. This was just done because the method came from elsewhere (another class) where it was referencing $field. This prevents having to do a find/replace for $field to $this.